train.py

```diff
@@ -100,8 +91,8 @@ def main(window_size, batch_size, ep_count, model_name, pretrained, debug):
                 break
 
             except:
-                logging.debug("Error connecting to Polygon, retrying in 30s (" + str(cnt) + "/30)")
-                time.sleep(30)
+                logging.warning("API timeout, reconnecting in 60s (" + str(cnt) + "/30)")
+                time.sleep(60)
```

Comment:
These changes are unrelated to the run file

Reply:
Oh, yeah, sorry about that, I use the free version of polygon.io directly, so I can only do 5 requests a minute.